### PR TITLE
fix: added error handling and removed logger print when fetching secrets

### PR DESCRIPTION
**Pull request by brynjarorng**
# ...

Attach a link to issue if relevant

## What

Better logs for feature deployment create db and stop printing the secret into console

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: infra/src/secrets.ts [hunk 1]
```diff
@@ -77,7 +100,11 @@ yargs(hideBin(process.argv))
 
       const { Parameter } = await ssm.getParameter(parameterInput).promise()
       if (Parameter) {
-        return logger.info(Parameter.Value)
+        if (Parameter.Value && Parameter.Value.length > 0) {
+          console.log(Parameter.Value)
```
Comment (lodmfjord) [inline infra/src/secrets.ts:104]: Do we want to console log what comes out of parameter store?

**Review by busla — APPROVED** [review]

File: infra/src/secrets.ts (hunk 1 above)
Comment (brynjarorng) [reply infra/src/secrets.ts:104]: In this case yes, If we use the logger, everything is always put into STD out which means that this export `a=$(node secrets get "$PGPASSWORD_KEY")` will ALWAYS print the output. Using console.log changes this behavior and will not print the secret

**Review by busla — CHANGES_REQUESTED** [review]
.

File: infra/src/secrets.ts (hunk 1 above)
Comment (busla) [reply infra/src/secrets.ts:124]: I dont follow. We need the value for the `create-db.sh` script

Comment (lodmfjord) [reply infra/src/secrets.ts:124]: thought this was something new was just checking the code

Comment (brynjarorng) [reply infra/src/secrets.ts:124]: Yes, the values is set in env in both cases. But using `console.log()` it is NOT printed in the console. If we use the logger, the env value is set AND it is always printed in plaintext to the console

Comment (lodmfjord) [reply infra/src/secrets.ts:124]: cool thanks for pointing out. I thought this was just debug log.

**Review by lodmfjord — APPROVED** [review]
LGTM

**Review by busla — APPROVED** [review]
LGTM

**Pull request merged**